README.md

```diff
@@ -202,6 +205,41 @@ DOJO_DOCKER_OPTIONS="--privileged"
 
 This method is suitable for running Dojo in Dojo. Dojo e2e tests use this method to test various Docker commands in a clean, separate Docker container. This way, such tests cannot affect Docker containers or images on Docker host (they have a safe environment to run in).
 
+### Public alpine image
+
+You can try any public Docker image. Example with an alpine image:
+```
+$ nano Dojofile
+$ cat Dojofile
+DOJO_DOCKER_IMAGE="alpine:3.16"
+$ dojo
+# now we run interactively in the Dojo Docker container
+/ # whoami
+root
+/ # pwd
+/
+/ # apk -h
+apk-tools 2.12.9, compiled for x86_64.
+```
+
+To exit the container, type `exit`.
+
+## Features
+
+You can use Dojo with any Docker image, but some features are only supported with custom Dojo Docker images.
+
+| Feature | Works with custom Dojo Docker images | Works with any image |
+| - | - | - |
+| Dojofile | Yes | Yes |
+| Starting, stopping, removing containers | Yes | Yes |
+| Signals support | Yes | Yes |
+| Support for running interactively and non-interactively | Yes | Yes |
+| All drivers support (Docker, Docker-compose) | Yes | Yes |
+| Preserving environment variables | Yes | Yes |
+| Custom startup scripts for a container | Yes | No |
+| Running as a custom user, not `root` | Yes | No |
+| Running in `/dojo/work` directory | Yes | No |
+| Ensuring proper UID and GID for custom user (to match file permissions) | Yes | No |
```

Comment:
The last one needs to be explained. Most people are not aware of the problem at all because they use osx and driver solves that.
I would call the feature "Handling permissions of files between host and container"

Reply:
And done :)